docs/developer-docs/latest/development/typescript.md

```diff
@@ -80,113 +80,29 @@ It is not necessary to repeat the `yarn install` or `npm run install` command af
 
 ## Start Strapi programmatically
 
-Instantiating Strapi programmatically in a TypeScript project requires additional configurations to load everything correctly. The primary difference for TypeScript programmatic use is that the codebase and compiled code are stored in separate directories, whereas the same directory is used to read and write in native JavaScript.
+Starting Strapi programmatically in a TypeScript project requires passing the output directory where the code is compiled.
 
-### Understand programatic use
+### Use the `strapi()` function
 
-When instantiating Strapi programmatically using the default export of `@strapi/strapi`, different parameters such as the `app` directory and the `dist` directory can be passed. The `app` directory is the project root directory and `dist` is a subdirectory of `app`. The `dist` directory represents the compiled project, with the same directory structure as the `app` directory, and the `app` directory represents the codebase (TypeScript or JavaScript). The main differences are:
-
-- When using the Content-type Builder to create, update, delete content types (or any other service that creates files), Strapi uses the `app` folder to write the files.
-- When reading what content types exist on the system Strapi reads the `dist` folder.
-- When the `develop` command is used only the `app` directory is watched. In TypeScript, a compilation will be triggered when a change is detected in the `app` directory and the output will be written to the `dist` directory.
-
-::: note
-The public folder is considered static and thus ignores the  `app` and `dist` directories.
-:::
-
-The default values for the `app` and `dist` directories are transformed and assigned using one of the following options:
+Strapi can be run programmatically by calling the `strapi()` function. Since the code of TypeScript projects is compiled in a specific directory, the parameter `distDir` should be passed to the `strapi()` function to indicate where the compiled code should be read:
```

Comment:
```suggestion
Strapi can be run programmatically by using the `strapi()` factory. Since the code of TypeScript projects is compiled in a specific directory, the parameter `distDir` should be passed to the factory to indicate where the compiled code should be read:
```